pkgs/development/libraries/clFFT/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, fftw, fftwFloat, boost166, opencl-clhpp, ocl-icd }:
+
+let
+  version = "2.12.2";
+in stdenv.mkDerivation {
+  pname = "clFFT";
```

```suggestion
  pname = "clfft";
```
https://nixos.org/nixpkgs/manual/#sec-package-naming